runtime/calamari/tests/integration_tests.rs

```diff
@@ -1108,8 +1108,7 @@ fn concrete_fungible_ledger_transfers_work() {
             ),);
 
             // Register and mint for testing.
-            // TODO:: Switch to u128::MAX when we start using https://github.com/paritytech/substrate/pull/11241
-            let amount = INITIAL_BALANCE;
+            let amount = u128::MAX;
```

Comment:
Wouldn't it be cleaner to access u128 through `calamari::runtime::types::Balance::MAX`?

Reply:
Actually, this is testcode, so nvm

Reply:
I can use the manta-primitives Balance, so it's more of an integration test.